guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md

```diff
@@ -255,7 +255,7 @@ The following tables give a detailed description of the results you can get usin
    </tbody>
 </table>
 
-### &lt;action&gt; {#fedg_layout_xml-instruc_ex_act}
```

Comment:
I would not eliminate the "<" and the ">" in this title, it references a node inside layout.